velox/serializers/PrestoSerializer.cpp

```diff
@@ -3069,6 +3071,183 @@ void estimateSerializedSizeInt(
   }
 }
 
+void flushUncompressed(
```

Comment:
I assume there is logic changes to compressed and uncompressed flush except function signature for the refactor? Thanks!